.travis.yml

```diff
@@ -1,26 +1,48 @@
-sudo: required
-dist: trusty
-
 language: node_js
-
-if: tag IS present
-
-before_install:
-  - cd webapp
+node_js:
+  "12"
 
 matrix:
   include:
     - os: linux
-
-node_js:
-  - "node"
+    - os: osx
+    - os: windows
+      filter_secrets: false # INSECURE: Workaround for issue https://travis-ci.community/t/current-known-issues-please-read-this-before-posting-a-new-topic/264/10
```

Comment:
filter_secrets filters travis secret from the build log. This was done because of the linked issue as it wouldn't even start the job otherwise. Since then, they changed it so that this property is set to false by default until this is fixed so I will remove it. See https://travis-ci.community/t/current-known-issues-please-read-this-before-posting-a-new-topic/264